.buildkite/scripts/github_token.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
what's this script about?

Reply:
you are right, forget delete this script after merging